standardguess

```diff
@@ -0,0 +1,382 @@
+        fi
+    else 
+        echo "${ELEMENT} already set to expected value of ${EXPECTATION}"
+    fi
+
+
+
+
+
+
+    echo
+    echo "Width"
+    _MKV_Prop_Check "${INPUT}" "Display width" "4"
+    _MKV_WIDTH "${INPUT}"
+
+    if [[ "${MKVWIDTH}" == "${ELEMENT} is not present" ]]; then
+        echo "Applying standard value to ${ELEMENT} of ${EXPECTATION}"
+        _APPLY_STANDARDS "${INPUT}" "display-width" "4"
+    elif [[ "${RETAIN}" == "1" ]]; then
+        echo "${ELEMENT} will retain the original value of ${SELECTED_ELEMENT_VALUE}"
+    elif [[ "${MKVWIDTH}" == "${ELEMENT} is not ${EXPECTATION} but it has the specified value of ${SELECTED_ELEMENT_VALUE}" ]] && [[ "${FORCE}" == "1" ]]; then
+        echo "${ELEMENT} is being reset from ${SELECTED_ELEMENT_VALUE} to ${EXPECTATION}"
+        _APPLY_STANDARDS "${INPUT}" "display-width" "4"
+    elif [[ "${MKVWIDTH}" == "${ELEMENT} is not ${EXPECTATION} but it has the specified value of ${SELECTED_ELEMENT_VALUE}" ]] && [[ "${FORCE}" != "1" ]]; then
+        read -e -p "Are you sure you would like to reset ${ELEMENT} to ${EXPECTATION}? [y/n] " choice
+        echo
+        if [[ "${choice}" == [Yy] ]]; then
+            echo "${ELEMENT} is being reset from ${SELECTED_ELEMENT_VALUE} to ${EXPECTATION}"
+            _APPLY_STANDARDS "${INPUT}" "display-width" "4"
+        else
+            echo "${ELEMENT} will retain the original value of ${SELECTED_ELEMENT_VALUE}"
+        fi
+    else 
+        echo "${ELEMENT} already set to expected value of ${EXPECTATION}"
+    fi
+
+
+
+
+
+    echo
+    echo "Height"
+    _MKV_Prop_Check "${INPUT}" "Display height" "3"
+    _MKV_HEIGHT "${INPUT}"
+
+    if [[ "${MKVHEIGHT}" == "${ELEMENT} is not present" ]]; then
+        echo "Applying standard value to ${ELEMENT} of ${EXPECTATION}"
+        _APPLY_STANDARDS "${INPUT}" "display-height" "3"
+    elif [[ "${RETAIN}" == "1" ]]; then
+        echo "${ELEMENT} will retain the original value of ${SELECTED_ELEMENT_VALUE}"
+    elif [[ "${MKVHEIGHT}" == "${ELEMENT} is not ${EXPECTATION} but it has the specified value of ${SELECTED_ELEMENT_VALUE}" ]] && [[ "${FORCE}" == "1" ]]; then
+        echo "${ELEMENT} is being reset from ${SELECTED_ELEMENT_VALUE} to ${EXPECTATION}"
+        _APPLY_STANDARDS "${INPUT}" "display-height" "3"
+    elif [[ "${MKVHEIGHT}" == "${ELEMENT} is not ${EXPECTATION} but it has the specified value of ${SELECTED_ELEMENT_VALUE}" ]] && [[ "${FORCE}" != "1" ]]; then
+        read -e -p "Are you sure you would like to reset ${ELEMENT} to ${EXPECTATION}? [y/n] " choice
+        echo
+        if [[ "${choice}" == [Yy] ]]; then
+            echo "${ELEMENT} is being reset from ${SELECTED_ELEMENT_VALUE} to ${EXPECTATION}"
+            _APPLY_STANDARDS "${INPUT}" "display-height" "3"
+        else
+            echo "${ELEMENT} will retain the original value of ${SELECTED_ELEMENT_VALUE}"
+        fi
+    else 
+        echo "${ELEMENT} already set to expected value of ${EXPECTATION}"
+    fi
```

Comment:
The above section seems to be re-used within the context of several different properties. It make reduce line count and make this easier to maintain if this section was adjusted to be a generic function which could be re-used within each context.